### ISSUE TITLE
NullReferenceException when starting an NFElectrical reactor

### ISSUE THREAD

**Issue by fat-lobyte** [ChrisAdderley/NearFutureElectrical#48]
Using RoverDude's MKS mod which also has nuclear reators, starting said reactor causes a Module

```
FissionReactor threw during OnFixedUpdate: System.NullReferenceException: Object reference not set to an instance of an object
  at NearFutureElectrical.FissionReactor.DoFuelConsumption () [0x00000] in <filename unknown>:0 
  at NearFutureElectrical.FissionReactor.OnFixedUpdate () [0x00000] in <filename unknown>:0 
  at Part.ModulesOnFixedUpdate () [0x00000] in <filename unknown>:0 
```
Removing NFElectricalUSI.cfg "fixes" the issue, but strips the reactor from NFE functionality.

## Attached Files

Full log file: [output_log.txt](https://github.com/ChrisAdderley/NearFutureElectrical/files/735180/output_log.txt)
Quicksave for easy testing: [nfe-use_nre.sfs.txt](https://github.com/ChrisAdderley/NearFutureElectrical/files/735181/nfe-use_nre.sfs.txt)
Here's a file listing for my GameData folder, excluding the Squad directory: [gamedata-filelist-nosquad.txt](https://github.com/ChrisAdderley/NearFutureElectrical/files/735201/gamedata-filelist-nosquad.txt)

## Installed Mods

- 000_USITools (0.3.7)
- B9PartSwitch
- CommunityResourcePack
- Firespitter
- NearFutureElectrical (0.8.3)
- UmbraSpaceIndustries (0.50.14)

## Steps to reproduce

1. Install MKS and NearFutureElectrical and their respective dependencies
2. Create and Launch a controllable craft with the MKS part "Nuclear Reactor (2.5m)" / alternatively, load quicksave file
3. In the right-click menu of the reactor, click "start reactor"
4. Observe Null Reference Exception
5. Turn the reactor off and then back on again
6. Observe no more Null Reference Exceptions

**Comment by fat-lobyte** [comment]
Apparently, this happens without MKS/USI installed as well.

Just take an NFE reactor and start it.

**Comment by ChrisAdderley** [comment]
I updated the title to reflect this, seems technically harmless at the moment

**Comment by ChrisAdderley** [comment]
Resolved in f6c6d614908efb6cd5b27cb8d2dac0d3127629c7
